extensions/resteasy-reactive/rest-client-reactive/deployment/src/main/java/io/quarkus/rest/client/reactive/deployment/RestClientReactiveProcessor.java

```diff
@@ -37,9 +41,13 @@
 import org.jboss.logging.Logger;
 import org.jboss.resteasy.reactive.common.processor.ResteasyReactiveDotNames;
 
+import com.google.common.collect.HashMultimap;
+import com.google.common.collect.Multimap;
```

Comment:
Is there a good reason to depend on these classes?

Reply:
Saves a few lines of code, we have the dependency on classpath, it's build-time only.
But I like to write code, I'm happy to replace multimap with code :)

Reply:
I'd really like it if we don't depend on Guava as much as possible :)